src/gtest/test_rpc.cpp

```diff
@@ -10,33 +10,6 @@
 #include "streams.h"
 #include "utilstrencodings.h"
 
-TEST(rpc, GetDifficultyTestnetRules) {
-    SelectParams(CBaseChainParams::TESTNET);
```

Comment:
Rather than just deleting this test, can we run the test for the mainnet parameters also with the testnet parameters?